app/src/docs/examples/dropdown/CloseOpenDelayModifiers.example.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { Dropdown, DropdownBodyProps } from '@epam/uui-components';
+import { Button, Text, FlexRow, DropdownContainer, FlexCell } from '@epam/promo';
+import * as css from './HandleStateExample.scss';
```

Comment:
could you please use default export instead? named exports will be disabled soon.
`import css from './HandleStateExample.scss'`